src/elements/content-sidebar/stories/tests/MetadataSidebarRedesign-visual.stories.tsx

```diff
@@ -96,6 +96,12 @@ export const FilterInstancesDropdown = {
                 expect(secondOption).toBeInTheDocument();
                 const thirdOption = canvas.getByRole('option', { name: 'Custom Metadata' });
                 expect(thirdOption).toBeInTheDocument();
+
+                await userEvent.click(firstOption);
+
+                const headlines = await canvas.findAllByRole('heading', { level: 4 });
+                expect(headlines).toHaveLength(1);
+                expect(headlines.map(heading => heading.textContent)).toEqual(expect.arrayContaining(['My Template']));
```

Comment:
am i missing something, if there is only one item in the `headlines`, do we have to use `map` here? cant we just assert against `headlines[0].textContent`?